.github/EXAMPLE_README_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@ This directory is a brief example of a [Name](site-link) site that can be deploy
 
 Deploy your own [Name] project with ZEIT Now.
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/now-examples/tree/master/example-directory)
+[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/import/project?template=https://github.com/zeit/now-examples/tree/master/example-directory)
```

Comment:
I still see 404

Reply:
Sorry @styfle, I'm on call with @leo right now and we're about to deploy and he asked me to make this ready for review

Reply:
Looks good now 👍